### PR TITLE
Tiddit subworkflow

### PR DESCRIPTION
**Pull request by WackerO**
This PR extracts tiddit from pair VC into a separate subworkflow

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
This superseded #637 then, right?

**Comment by WackerO** [comment]
> This superseded #637 then, right?

I would suggest first merging #637 and then merging this PR separately, I was going to work on any SVDB errors on another branch

**Comment by FriederikeHanssen** [comment]
but you are including tabix in both?

**Comment by WackerO** [comment]
Well yes, #637 introduces it and 651 moves it together with the rest of TIDDIT. Or should the indexing still be done in pairVC, not in the new tiddit subworkflow?

**Comment by WackerO** [comment]
On it. Took me a bit to find out how what I needed to change in the modules.config. I hope all the tests run...

**Comment by WackerO** [comment]
fiiinally tiddit seems to work. I had to re-enable the vcf,vcf.gz though (from your last review) or the pytest would throw errors...

File: conf/modules.config [hunk 1]
```diff
@@ -1091,21 +1091,21 @@ process{
...
 
         //SVDB
-        withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:SVDB_MERGE' {
+        withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_TIDDIT_SOMATIC:SVDB_MERGE' {
             ext.prefix       = { "${meta.id}.tiddit" }
             publishDir       = [
                 mode: params.publish_dir_mode,
                 path: { "${params.outdir}/variant_calling/tiddit/${meta.id}/" },
-                pattern: "*vcf"
+                pattern: "*{vcf,vcf.gz}"
```
[the remark this answers is not in the document]
Comment (FriederikeHanssen) [reply conf/modules.config:1108]: ah right this needs to stay until the mulled container is here

**Pull request merged**